src/views/HashtagPostView/index.tsx

```diff
@@ -81,11 +84,21 @@ const HashtagPostView = ({ pageData, seo }: HashtagProps): JSX.Element => {
       <Container py={4}>
         <Box textAlign="center">
           <Heading>{pageData?.hashtag?.title}</Heading>
-          <Text my={4} maxW="container.md" mx="auto">
-            {pageData?.hashtag?.content}
-          </Text>
+          <Collapse startingHeight={35} in={show}>
+            <Text my={4} maxW="container.md" mx="auto">
+              {pageData?.hashtag?.content}{' '}
+            </Text>
+          </Collapse>
+          <Button
+            bg="transparent"
+            borderColor="transparent"
+            color="primary.500"
+            size="sm"
+            onClick={handleToggle}
+          >
+            {show ? 'Less' : 'More'}
+          </Button>
```

Comment:
Just a suggestion:

We can use `<IconButton icon={FaChevronDown} />` instead of text button. In this case we don't need to worry about button text translation.